extension-manifest-v3/src/store/tab-stats.js

```diff
@@ -14,7 +14,10 @@ import { store } from 'hybrids';
 const Tracker = {
   id: true,
   name: '',
-  requests: [{ url: '', blocked: false }],
+  requests: [{ url: '', blocked: false, modified: false }],
+  requestsBlocked: ({ requests }) => requests.filter((r) => Boolean(r.blocked)),
+  requestsModified: ({ requests }) =>
+    requests.filter((r) => Boolean(r.modified)),
```

Comment:
The `Boolean` function wrapper is not required here, and below, the filter function maps to boolean internally (by checking truthy/falsy value.